LiteCore/tests/LazyVectorAPITest.cc

```diff
@@ -407,6 +407,7 @@ TEST_CASE_METHOD(LazyVectorAPITest, "BeginUpdate on Non-Vector", "[API][.VectorS
 
     C4Error err{};
     auto    _ = c4index_beginUpdate(index, 10, &err);
+    (void)_;
```

Comment:
```suggestion
    CHECK(c4index_beginUpdate(index, 10, &err) == nullptr);
```
Same for the instance below.
This is just basic error checking, guys... 😩

Reply:
This is because our build checks are less stringent than Xcode. This is the first thing I hit after pulled from the remote and quickly got it over. 
Will do that check